src/store/ChartStore.js

```diff
@@ -247,7 +248,15 @@ class ChartStore {
                 } else {
                     this.changeSymbol(this.defaultSymbol);
                 }
+
                 this.setLayoutData(context);
+                /**
+                 * Updating market close status each 10 minute
+                 */
+                this.updateMarketClosedStatus();
+                setInterval(() => {
+                    this.updateMarketClosedStatus();
+                }, 10 * 60 * 1000);
```

Comment:
@shayanbinary nitpick: `setInterval(this.updateMarketClosedStatus, 10 * 60 * 1000)`;